packages/arb-bridge-eth/contracts/inbox/GlobalInbox.sol

```diff
@@ -117,6 +113,31 @@ contract GlobalInbox is
         _deliverMessage(chain, L2_MSG, msg.sender, messageData);
     }
 
+    function deployL2ContractPair(
+        address chain,
+        uint256 maxGas,
+        uint256 gasPriceBid,
+        uint256 payment,
+        bytes calldata contractData
+    ) external {
+        require(isContract(msg.sender), "must be called by contract");
+        requestPairing(msg.sender, chain);
+        _deliverMessage(
+            chain,
+            L2_CONTRACT_PAIR,
+            msg.sender,
+            abi.encodePacked(
```

Comment:
Flagging that this message format is changing.  Probably good to incorporate the new format into this PR?